### PR TITLE
update vulnerabilities

### PR DESCRIPTION
**Pull request by justincorrigible**
The blog's build pipeline has been broken for a few months.
These changes should fix that.

Testable at https://jekyll-update--oicr-softeng.netlify.app/, after this build log https://app.netlify.com/sites/oicr-softeng/deploys/5f9afef607830d2c3938284f

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -106,4 +108,4 @@ DEPENDENCIES
   kramdown (= 1.13.2)
 
 BUNDLED WITH
-   2.1.4
+   2.0.1
```
Comment (justincorrigible) [inline Gemfile.lock:111]: This version is compatible with what Netlify offers out of the box.

**Comment by rosibaj** [comment]
@caravinci can we close or merge this PR?

**Comment by justincorrigible** [comment]
Gladly! As soon as someone reviews+approves it. I'd much rather we don't just close it, though, as it addresses vulnerabilities; but I guess that's an option too 👍 
Thoughts?

File: LICENSE [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Copyright 2016(c) The Ontario Institute for Cancer Research. All rights reserved.
+Copyright 2020(c) The Ontario Institute for Cancer Research. All rights reserved.
```
Comment (joneubank) [inline LICENSE]: Do we want this to be 2021 now?

**Review by joneubank — APPROVED** [review]

**Pull request merged**